docs/csharp/programming-guide/concepts/linq/how-to-modify-an-office-open-xml-document.md

```diff
@@ -18,7 +18,7 @@ ms.author: "wiwagn"
 # How to: Modify an Office Open XML Document (C#)
 This topic presents an example that opens an Office Open XML document, modifies it, and saves it.  
   
- For more information on Office Open XML, see [www.openxmldeveloper.org](http://go.microsoft.com/fwlink/?LinkID=95573).  
+ For more information on Office Open XML, see [www.ericwhite.com](http://ericwhite.com/).  
```

Comment:
It might be useful to add a link to the Office Open XML SDK at GitHub.com/OfficeDev/Open-XML-SDK